integration/run_test.go

```diff
@@ -353,6 +354,41 @@ var _ = Describe("Running Specs", func() {
 		})
 	})
 
+	Context("when pointed at a packages with a failing table test", func() {
+		BeforeEach(func() {
+			pathToTest = tmpPath("failing_table_tests")
+			copyIn(fixturePath("failing_table_tests"), pathToTest, false)
+		})
+
+		It("should identify the failing entry", func() {
+			session := startGinkgo(pathToTest, "--noColor")
+			Eventually(session).Should(gexec.Exit(1))
+			output := string(session.Out.Contents())
+
+			Ω(output).Should(ContainSubstring("Test Suite Failed"))
+			Ω(output).Should(ContainSubstring(fmt.Sprintf("%s Failure", denoter)))
+
+			Ω(output).Should(ContainSubstring("Summarizing 1 Failure:"))
+			Ω(output).Should(ContainSubstring("[Fail] FailingTableTests a failing entry [It] failing"))
+
+			// The table
+			Ω(output).Should(
+				ContainSubstring("failing_table_tests/failing_table_tests_test.go:11"),
+				"point to the failing table")
+			Ω(output).ShouldNot(MatchRegexp(`github.com/onsi/ginkgo/extensions/table/table.go:\[\d+\]`))
+
+			// The entry
+			Ω(output).Should(
+				ContainSubstring("failing_table_tests/failing_table_tests_test.go:14"),
+				"point to the failing entry")
+			Ω(output).ShouldNot(MatchRegexp(`github.com/onsi/ginkgo/extensions/table/table_entry.go:\[\d+\]`))
+
+			Ω(output).Should(
+				ContainSubstring("failing_table_tests/failing_table_tests_test.go:12"),
+				"point to the failing assertion")
```

Comment:
It's cool that you provide the optional test description given the complexity of the test.